packages/blade/src/utils/makeAccessible/accessibilityMapNative.ts

```diff
@@ -97,4 +97,5 @@ export const accessibilityRoleMap = {
   keyboardkey: 'keyboardkey',
   label: 'label',
   text: 'text',
+  meter: 'progressbar',
```

Comment:
Should we map the `meter` role to progressbar on native? Not sure if every static meter should be treated as a progressbar 🤔

Reply:
There's no other similar role available on native. I tried looking up what we could do but no luck. Progress bar is the closest to the use-case here

Reply:
Not much expertise on native a11y, perhaps we don't add a role for this? Just works as a regular UI.